bootstrap/widgets.py

```diff
@@ -73,7 +74,7 @@ def build_attrs(self, *args, **kwargs):
         return attrs
 
 
-class TextInput (BootstrapWidget, forms.TextInput):
+class TextInput (apps.get_app_config('bootstrap').get_bootstrap_widget(), forms.TextInput):
```

Comment:
I think we can just import the class at the top of the file and use it in these classes below. (I'm not entirely sure what to name it since BootstrapWidget  already exists in this file... so feel free to change the name if you can think of a better one)

```
MainBootstrapWidget = apps.get_app_config('bootstrap').get_bootstrap_widget()

class TextInput (MainBootstrapWidget, forms.TextInput)
class PasswordInput (MainBootstrapWidget, forms.PasswordInput)
```